tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -3167,8 +3172,148 @@ namespace EnergyPlus {
 				"12,11,Electricity:Facility [J] !RunPeriod [Value,Min,Month,Day,Hour,Minute,Max,Month,Day,Hour,Minute]",
 			} ) );
 
+			TimeValue.deallocate(); 
```

Comment:
@JasonGlazer This is not needed. The TearDown() phase of the Fixture will take care of it. Plus it is added to another unit test.